presentations/rte-workshop.md

```diff
@@ -0,0 +1,356 @@
+class: center, middle, intro
+
+# Remote Testing Environment Workshop
+
+### OSFC 2018
+
+## Piotr Król, Michał Żygowski
+
+<img src="images/logo.png" width="150px" style="margin-left:-20px">
+
+---
+
+# Agenda
+
+* Introduction
+* Firmware developer routines
+* Remote Testing Environment
+* Infrastructure
+* Hardware setup
+* Automation Tests
+* Fixing bugs
+* Future plans
+    - RTE Control API (beta)
+    - New RTE revision for RPi
+* Q&A and RTE giveaway
+
+---
+
+# Introduction
+
+.center[
+.left-column50[<img src="images/piotr_krol.jpg" width="200px">]
+.right-column50[<img src="images/michal_zygowski.png" width="200px">]
+
+.left-column50[__Piotr Król__]
+.right-column50[__Michał Żygowski__]
+.left-column50[_Founder & Embedded Systems Consultant_]
+.right-column50[_Firmware Engineer_]
+]
+
+.right-column50[
+<br/>
+- <img src="images/email.png" width="24px"/> michal.zygowski@3mdeb.com
+- <img src="images/linkedin.png" width="24px"/>[linkedin.com/in/michał-żygowski-88954416b](https://www.linkedin.com/in/michał-żygowski-88954416b/)
+]
+
+.left-column50[
+- <img src="images/twitter.png" width="24px"/> @pietrushnic
+- <img src="images/email.png" width="24px"/> piotr.krol@3mdeb.com
+- <img src="images/linkedin.png" width="24px"/> [linkedin.com/in/krolpiotr](https://www.linkedin.com/in/krolpiotr)
+- <img src="images/facebook.png" width="24px"/>[facebook.com/piotr.krol.756859](https://www.facebook.com/piotr.krol.756859)
+]
+
+---
+
+# Firmware developer routines
+
+* Narrowing down serious issue requires a lot of repetitive work:
+<br/><br/>
+    - modify & build firmware with potential fix
+    - flash affected firmware version
+    - boot system
+    - verify if the bug still exists
+<br/><br/>
+* Problem with automation and remote development
+
+<br/>
+To get rid of these problems we created Remote Testing Environment.
+
+---
+
+# Remote Testing Environment
+
+.center[<img src="images/rte-opi.png" width="650px">]
```

Comment:
@pietrushnic @artur-rs didn't we mention about better contrast on this picture? The lines were not visible enough to see where they point.

Reply:
@miczyg1 I have changed the image for the one with transparent background